AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Main_Null_Render_MaterialCanvas_01.py

```diff
@@ -0,0 +1,30 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import logging
+import pytest
+
+from ly_test_tools.o3de.atom_tools_test import AtomToolsBatchedTest, AtomToolsTestSuite
+from ly_test_tools.launchers import launcher_helper
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+@pytest.mark.parametrize("launcher_platform", ['windows_material_canvas'])
+class TestMaterialCanvas(AtomToolsTestSuite):
+
+    log_name = "material_canvas_test.log"
+    executable_function = launcher_helper.create_material_canvas
```

Comment:
No because the MaterialEditor tests are the default. It can still be explicitly set without issue so its no cause for concern either way on the user end.

Reply:
In the next update I will make sure these get set explicitly